### PR TITLE
Support for storage type config

### PR DESCRIPTION
**Pull request by mrkaspa**
@stavro @baoist check plz this

### CODE REVIEW

**Comment by vitalis** [comment]
:+1:

**Comment by vitalis** [comment]
@mrkaspa is `uploads/image.png` should be a part of this pull request ?

File: config/config.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+config :arc,
+  arc_storage: Arc.Storage.S3
```
Comment (nathany) [inline config/config.exs]: I would suggest just calling this key `storage` rather than `arc_storage`.

**Comment by nathany** [comment]
This looks like it could resolve #32, which would be really nice.

**Comment by dotdotdotpaul** [comment]
This is a really key feature, is there any good reason why this isn't getting pulled in or otherwise added to the mainline?

**Comment by philipgiuliani** [comment]
This is a very nice feature. Can we get it merged?

**Comment by stavro** [comment]
Closed via https://github.com/stavro/arc/commit/848c17e5d51e658e10e6824cb4e97bab9e871456

**Pull request closed without merging**